<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debit</class>
 <widget class="QMainWindow" name="Debit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debit calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(38, 38, 38);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="ckb_capital">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Капитализация процентов</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_refill">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Пополнения</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_withdrawal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Частичные снятия</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>340</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlo_sum">
     <item>
      <widget class="QLabel" name="lbl_sum">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Сумма вклада</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spn_sum">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_sum_rub">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>руб.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>336</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlo_limit">
     <item>
      <widget class="QLabel" name="lbl_limit">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Срок размещения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spn_limit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_limit">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>дней</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>месяцев</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>лет</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>260</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlo_date">
     <item>
      <widget class="QLabel" name="lbl_date">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Начало срока</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dt_start">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>340</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlo_percent">
     <item>
      <widget class="QLabel" name="lbl_percent_f">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Процентная ставка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spn_percent">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_percent_b">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>% годовых</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>306</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlo_period">
     <item>
      <widget class="QLabel" name="lbl_period">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Переодичность выплат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_period">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>каждый день</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>каждую неделю</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>раз в месяц</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>раз в квартал</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>раз в год</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>в конце срока</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
